pkg/kubectl/cmd/helpers.go

```diff
@@ -26,7 +27,10 @@ import (
 	"strings"
 	"time"
 
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
 	"github.com/golang/glog"
+	"github.com/imdario/mergo"
```

Comment:
No, it's actually already there.  looks like @deads2k added it with the config changes:

https://github.com/GoogleCloudPlatform/kubernetes/tree/master/Godeps/_workspace/src/github.com/imdario/mergo